### ISSUE TITLE
Old bot comments are not retired yet

### ISSUE THREAD

**Issue by crimsonfaith91** [kubernetes/test-infra#3462]
The old bot comments `@k8s-bot test this`, `@k8s-bot <some-test-name> test this` and `@k8s-bot ok to test` are still working. We should retire them.

**Comment by kargakis** [comment]
They are still working I guess because k8s-merge-bot is using them to rerun tests for PRs with old test results.

**Comment by crimsonfaith91** [comment]
@kargakis Thanks for the information! I will look at how k8s-merge-bot uses them.

**Comment by cblecker** [comment]
I think it's this message that needs to be fixed:
```
@k8s-bot test this

Tests are more than 96 hours old. Re-running tests.
```

**Comment by crimsonfaith91** [comment]
@cblecker Noted! Thanks! :)

**Comment by xiangpengzhao** [comment]
Fixed by #3845
/close
